app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +32,13 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    protected function authenticated(Request $request, $user)
+    {
+        Login::create([
+            'user_id' => $user->id,
+            'user_agent' => $request->header('User-Agent')
+        ])->save();
+        return redirect()->route('home');
```

Comment:
I suspect this wants to redirect to `/` as the app doesn't currently have a home route.

Reply:
route recives name  of the route as parameter and you have `Route::get('/', 'CampaignController@index')->name('home');` in your web.php